todo_app/templates/index.html

```diff
@@ -4,7 +4,7 @@
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">Just another to-do app, but automated thought github actions, docker hub and azure!</p>
```

Comment:
Very minor, but I think a typo here! 😄 
```suggestion
    <p class="lead">Just another to-do app, but automated through github actions, docker hub and azure!</p>
```